assets/js/util/tracking/createTrackEvent.js

```diff
@@ -61,9 +61,11 @@ export default function createTrackEvent(
 			dimension1: referenceSiteURL,
 			dimension2: currentUserRoles.join( ',' ),
 			dimension3: userIDHash,
-			dimension4: global.GOOGLESITEKIT_VERSION || '',
+			dimension4: pluginVersion || '',
 			dimension5: Array.from( enabledFeatures ).join( ',' ),
 			dimension6: activeModules.join( ',' ),
+			// Custom dimension to differentiate between authenticated and non authenticated users.
```

Comment:
I know the IB mentions this but I'm not sure the comment adds much value as it reads to be rather self-documenting IMO 😄

Reply:
😁 Removed!